Compile error at comments.go

Import "net/url" in `comments.go` not used, causing compiler to stop with error: 
https://github.com/ahmdrz/goinsta/blob/master/comments.go#L6

```
$ go build
# github.com/ahmdrz/goinsta/v2
..\..\..\..\pkg\mod\github.com\ahmdrz\goinsta\v2@v2.4.1-0.20190305073421-37afbe8ea07e\comments.go:6:2: imported and not used: "net/url"
```

--- a/comments.go
+++ b/comments.go
@@ -3,7 +3,6 @@ package goinsta
 import (
 	"encoding/json"
 	"fmt"
-	"net/url"
 	"strconv"
 )
 

